Polyclinic/Polyclinic.Infrastructure.InMemory/DoctorInMemoryManager.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Polyclinic.Domain;
+
+namespace Polyclinic.Infrastructure.InMemory;
+
+/// <summary>
+/// In-memory implementation of the manager for doctors providing CRUD operations
+/// </summary>
+public class DoctorInMemoryManager : IManager<Doctor, int>
+{
+    private readonly List<Doctor> _doctors;
+
+    /// <summary>
+    /// Initializes a new instance of the DoctorInMemoryManager class with pre-seeded test data
+    /// </summary>
+    public DoctorInMemoryManager()
+    {
+        _doctors = TestDataSeeder.GetDoctors();
+    }
```

Comment:
Можно избавиться от конструктора сократив код. По аналогии в других классах репозиториев.

```suggestion
public class DoctorInMemoryManager : IManager<Doctor, int>
{
    private readonly List<Doctor> _doctors = TestDataSeeder.GetDoctors();
```